spec/helpers/atmosphere/funds_helper_spec.rb

```diff
@@ -37,7 +37,7 @@
       expect(last_months_names.last).
         to eq Date::MONTHNAMES[Time.now.month]
       expect(last_months_names.first).
-        to eq Date::MONTHNAMES[(Time.now.month + 1) % 12]
+        to eq Date::MONTHNAMES[(Time.now + 1.month).month]
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.